src/channel-picker/channel-picker.js

```diff
@@ -25,14 +25,14 @@ const Self = class ChannelPicker extends ColorElement {
 	connectedCallback () {
 		super.connectedCallback?.();
 
-		this._el.picker.addEventListener("change", this);
+		this._el.picker.addEventListener("input", this, {capture: true});
```

Comment:
input usually bubbles. We shouldn't need capture. If we do, we are emitting an input somewhere and forgot to bubble it.

Reply:
You are absolutely right. Fixed. Thank you!